lib/kernel.nix

```diff
@@ -16,9 +16,16 @@ in
   unset    = { tristate    = null; optional = false; };
   freeform = x: { freeform = x; optional = false; };
 
-  /*
+  /**
     Common patterns/legacy used in common-config/hardened/config.nix
-   */
+
+
+    # Inputs
+
+    `version`
+
+    : 1\. Function argument
+  */
```

Comment:
This one seems like an implementation comment. What do you think about reverting it to normal comments?